### PR TITLE
Project pages

### PR DESCRIPTION
**Pull request by TyHil**
- [x] Jupiter
- [x] Trends
- [x] Skedge
- [x] API
- [x] Planner

### CODE REVIEW

File: src/pages/projects/index.tsx
Comment (nl32) [inline src/pages/projects/index.tsx:1]: now that we have pages for each project I'm not sure how necessary the overview page is.

File: src/components/Navbar.tsx [hunk 1]
```diff
@@ -58,36 +60,36 @@ const parentItems: Array<ParentItem> = [
...
-        link: 'https://chromewebstore.google.com/detail/skedge/ghipfanpcodcmkjacmmfjdmccdiaahab',
+        link: '/projects/skedge',
         description: 'Integrate grade and professor stats into Schedule Planner',
         iconSrc: TrendingUp,
       },
       {
         name: 'API & Platform',
-        link: 'https://github.com/UTDNebula/nebula-api',
+        link: '/projects/api',
         description: 'Integrate X+ years of historical UTD data into your applications',
```
Comment (nl32) [inline src/components/Navbar.tsx:84]: We can open a new issue for this but should probably have an actual number instead of an x.

**Review by nl32 — APPROVED** [review]
Some feedback but the project pages look amazing.

File: src/pages/projects/index.tsx
Comment (TyHil) [reply src/pages/projects/index.tsx:1]: Good point, I'll remove it

File: src/components/Navbar.tsx (hunk 1 above)
Comment (TyHil) [reply src/components/Navbar.tsx:84]: I asked Josh, pretty sure it's since 2017

**Pull request merged**